tests/Files.InteractionTests/SessionManager.cs

```diff
@@ -70,13 +70,13 @@ public static void CreateSession(TestContext _)
 				if (_session is null)
 				{
 					// WinAppDriver is probably not running, so lets start it!
-					if (File.Exists(@"C:\Program Files (x86)\Windows Application Driver\WinAppDriver.exe"))
+					if (File.Exists($@"{Environment.GetEnvironmentVariable("SystemDrive")}\Program Files (x86)\Windows Application Driver\WinAppDriver.exe"))
```

Comment:
Use %ProgramData(x86)%
```suggestion
					if (File.Exists($@"{Environment.GetEnvironmentVariable("ProgramData(x86)")}\Windows Application Driver\WinAppDriver.exe"))
```

Reply:
I've reverted the change here since this is not an app code but a test code.

Reply:
But Interaction Tests can be run on local.

Reply:
> But Interaction Tests can be run on local.

True, I changed the code again.